changelog.md

```diff
@@ -2,6 +2,10 @@
  ### Add-ons client
  ### Add-ons server
  ### Campaigns
+   * Eastern Invasion
+     * S07a: 
+       * Hide side 2 without spoiling it (issue #4145)
```

Comment:
Need to be wary of the changelog being a spoiler too - maybe "better handling for side 2"?